### ISSUE TITLE
jaws module create fails

### ISSUE THREAD

**Issue by jro** [jaws-framework/JAWS#132]
I just setup, so this might be a problem specific to my environment. But if I try to create a new module I get an error re: missing "download" module. I've checked and it should be installed globally.

    $ jaws module create tacos new
    module.js:338
        throw err;
        ^
    
    Error: Cannot find module 'Download'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/usr/local/lib/node_modules/jaws-framework/lib/commands/module_install.js:16:16)
    ...

**Comment by jodagm** [comment]
+1
I get the same thing, 
Download = require('Download');
should change to:
Download = require('download');

**Comment by doapp-ryanp** [comment]
Fixed. Thanks. https://github.com/jaws-framework/JAWS/commit/56bc47ea009807947d3fb9cb496f2e32d997e1a8
